purchasing/settings.py

```diff
@@ -55,6 +55,7 @@ class ProdConfig(Config):
     CACHE_TYPE = 'redis'
     CACHE_REDIS_URL = os_env.get('REDIS_URL', 'redis://localhost:6379/0')
     CACHE_DEFAULT_TIMEOUT = 30
+    EXTERNAL_LINK_WARNING = False
```

Comment:
I would just make it be configurable from the environment, see the comment on the PR below.